ega-charts/cega/conf/cega.json

```diff
@@ -1,7 +1,6 @@
 {"rabbit_version":"3.6.14",
     "users":[
-        {"name":"lega","password_hash":"4FALlX+/iXbxXHtAx7usIz+Q6MksLe1CVKDEIqX11mfIbGB6",
-        "hashing_algorithm":"rabbit_password_hashing_sha256","tags":"administrator"},
+        {"name":"lega","password":"lega","tags":"administrator"},
```

Comment:
this cega.json can be generated with https://github.com/NBISweden/LocalEGA-deploy-init and it is good to realise that in CEGA probably will not accept clear passwords, so I would recommend generating that. If unsure how to do it there this: https://github.com/NBISweden/LocalEGA-deploy-init/blob/master/lega_init/configure.py#L148-L164 or this: https://github.com/EGA-archive/LocalEGA/blob/master/extras/rabbitmq_pwhash.py

Reply:
seems like @jbygdell is doing something similar in https://github.com/NBISweden/LocalEGA-helm/pull/18

Reply:
I'm all for generating this stuff. I was just unable to work out what that hash actually corresponded when I was configuring the mq secrets. It'd be nice if we could just import the values directly into helm with an additional `-f secrets.yml` param or something. It looks like you're exporting this stuff as yaml anyway so shouldn't be too hard to automate/document.

Reply:
The init script generates `cega_mq_pass` which corresponds to the hashed password in the `cega.json` file.
The biggest problem with helm is that files other than `values.yaml` must be located inside the helm tree structure, as such it is not possible to load files from external folders.

But in #18 the passwords for the fake cega environment are replaced with `fake_cega`, mostly to simplify testing in our CI environment.